ui/pages/onboarding-flow/create-password/create-password.js

```diff
@@ -126,7 +129,7 @@ export default function CreatePassword({
       return;
     }
     // If secretRecoveryPhrase is defined we are in import wallet flow
-    if (secretRecoveryPhrase) {
+    if (secretRecoveryPhrase && firstTimeFlowType === 'import') {
```

Comment:
[nit] we could create an enum style object for `firstTimeFlowTypes`:

```
const firstTimeFlowTypes = {
IMPORT = 'import',
CREATE = 'create'
}
```

Reply:
This could be added on `onboarding-flow.js` and imported and used all the spots where we use `firstTimeFlowType === 'import'` here.

Reply:
I did this, but in a different way. I created a new file called `onboarding.js` within the `ui/helpers/constants` folder, as there are the other constants, and there I put this newly created object and exported it from there.
You can see the changes I made here: d3aaacb8d1e620485355883185ba645e858a63e1